packages/phoenix-client/src/phoenix/client/resources/spans/__init__.py

```diff
@@ -403,19 +503,21 @@ async def get_span_annotations_dataframe(
         *,
         spans_dataframe: Optional["pd.DataFrame"] = None,
         span_ids: Optional[Iterable[str]] = None,
+        spans: Optional[list[Span]] = None,
```

Comment:
Can use `Iterable` to be more general.
```suggestion
        spans: Optional[Iterable[Span]] = None,
```